dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/MainWindow.xaml.cs

```diff
@@ -63,11 +64,33 @@ private void CustomerGrid_SelectionChanged(object sender, System.Windows.Control
             barGraphHost.DataContext = (Customer)CustomerGrid.SelectedItem;
         }
 
+        private static Customer GenerateRandomCustomer()
+        {
+            Customer c = new Customer
```

Comment:
Just return new Customer {...} - the c varible is redundant.

Reply:
updated